content/en/blog/_posts/2020-05-xx-third-party-dual-sourced-content.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "How Docs Handle Third Party and Dual Sourced Content"
+date: 2020-05-xx
+slug: third-party-dual-sourced
+url: /blog/2020/05/third-party-dual-sourced-content
+---
+
+**Author:** Zach Corleissen | [GitHub](https://github.com/zacharysarah/) | [Twitter](https://twitter.com/zachorsarah)
```

Comment:
I expected to see:
```suggestion
**Author:** Zach Corleissen, Cloud Native Computinf Foundation
```

Reply:
I'll incorporate the idea but not the spelling error. 😆

Reply:
D'oh!